### DISCUSSION TITLE
My account is disabled since last more than 15 days

### DISCUSSION THREAD

**Discussion by multidevelopers** [community/community#134928] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

My account is disabled since last more than 15 days without any warning and with no reason remarks.
Also submitted support tickets for this issue. but there is no any response from github support team.
My all projects was uploaded on that account, now I am completely handi cap due to this issue.
Last submitted support ticket id is: 2905940

**Comment by queenofcorgis** [comment, marked as answer]
Hi @multidevelopers, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

_Please note for future posts:  You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
